lib/config/definitions.js

```diff
@@ -95,23 +95,23 @@ const options = [
     name: 'commitMessage',
     description: 'Commit message template',
     type: 'string',
-    default: 'Update dependency {{depName}} to version {{newVersion}}',
+    default: 'Update dependency {{depName}} to version {{newVersionRange}}',
```

Comment:
Please see this comment about the use of `newVersionRange` - https://github.com/singapore/renovate/pull/139#pullrequestreview-31407067

It was simply to work around how `semver` works. I have no preference on naming scheme.